docs/ref/built-in-functions.md

```diff
@@ -26,3 +26,6 @@
 
 - **idadi(`kitu: SWList | SWString`): `SWNumber`**<br/>
   Returns the length of a list or string
+
+- **tarehe(`siku: SWDateTime| siku: SWDateTime, "kitu": SWString`): `SWDateTime`** <br/>
```

Comment:
To indicate optional params, use square brackets around the def like this:

```suggestion
- **tarehe(`siku: SWDateTime, [muundo: SWString]`): `SWDateTime`** <br/>
```

Reply:
Ah, didn't know that. Thanks